README.md

```diff
@@ -58,4 +59,5 @@ jobs:
       - name: Resolve packages
         run: npm install
       - name: Deploy stack to aws
-        run: nitric up -s dev -v0
\ No newline at end of file
+        run: nitric up -s dev -v0
```

Comment:
once this https://github.com/nitrictech/cli/pull/205 is released, we can change this to "nitric up -s dev --ci"

Reply:
This has now been released.